enterprise_access/apps/api/v1/tests/test_allocation_view.py

```diff
@@ -6,8 +6,10 @@
 
 import ddt
 from django.core.cache import cache as django_cache
+from django.core.exceptions import ValidationError
 from rest_framework import status
 from rest_framework.reverse import reverse
+from rest_framework.serializers import ValidationError
```

Comment:
@iloveagent57  can you please look into these changes again? these are [failing the tests](https://github.com/openedx/enterprise-access/actions/runs/6724324708/job/18276322825?pr=313#step:7:407). IT's also causing my PR fail.

Reply:
@zubairshakoorarbisoft FWIW, I also ran into this issue with my [PR](https://github.com/openedx/enterprise-access/pull/314). I made the changes requested by `make quality` (i.e., removing this duplicate import and removing an unnecessary pylint disable) alongside my PR's changes to get CI to pass; you could try doing similar in your PR.

Reply:
Alright, Thanks @adamstankiewicz

Reply:
whoops, sorry about that, folks!  Looks like Adam's PR cleaned that up and was merged.